db-backup/Dockerfile

```diff
@@ -1,10 +1,22 @@
-FROM python:3.6-slim
+FROM python:3.6.12-slim
```

Comment:
Any reason not to go to `python:3.6.13-slim` now?

Reply:
👀  good spot @alex9smith updated now in 79cd77a